src/tox_travis/after.py

```diff
@@ -81,10 +80,9 @@ def travis_after():
     print('All required jobs were successful.')
 
 
-def after_config_matches():
+def after_config_matches(cfg):
     """Determine if this job should wait for the others."""
-    config = py.iniconfig.IniConfig('tox.ini')
-    section = config.sections.get('travis:after', {})
+    section = cfg.sections.get('travis:after', {})
```

Comment:
I agree that's its a bit annoying to pass the config down the stack, but I also agree that I'm not convinced that the other options are clear improvements at this point. If we had other global(ish) state that we needed, then `ToxAfter` (and probably a matching `ToxEnv`) would probably be my preferred approach. As it is, we'd probably just end up trading `config` in several places with `self` everywhere and extra indentation.